docs/weaver_code_anatomy.md

```diff
@@ -62,25 +59,35 @@ code: |
   if not defined("interview_metadata['a_258e_motion_for_impoundment']"):
     interview_metadata.initializeObject('a_258e_motion_for_impoundment')
   interview_metadata['a_258e_motion_for_impoundment'].update({
-    'title': '209a 258e motion for impoundment',
-    'short title': '209a 258e motion for impoundment',
-    'description': '209a 258e motion for impoundment',
-    'original_form': '',
-    'allowed courts': [
-      'Land Court',
+    "al_weaver_version": "0.82",
+    "generated on": "2021-05-28",
+    "title": "209a 258e motion for impoundment",
+    "short title": "209a 258e motion for impoundment",
+    "description": "209a 258e motion for impoundment",
+    "original_form": "",
+    "allowed courts": [
+      "Land Court",
     ],
-    'categories': [
-      'Guardianship',
+    "categories": [
+      "BE-04-00-00-00",
     ],
-    'logic block variable': 'a_258e_motion_for_impoundment',
-    'attachment block variable': 'a_258e_motion_for_impoundment_attachment',
-    'typical role': 'plaintiff',
+    "logic block variable": "a_258e_motion_for_impoundment",
+    "attachment block variable": "a_258e_motion_for_impoundment_attachment",
+    "typical role": "plaintiff",
   })
 ---
 ```
 
+1. `title`, `short title`, and `description` allow your organization's site to show more information about your form and to organize your forms more easily.
+1. `original_form` is simply a link to the original, fillable PDF form that this interview is automating, if it exists.
+1. 🚧 `allowed courts` allows your code to decide which courts to let the user pick from when they need to pick their court, usually used in conjunction with [AL Generic Jurisdiction](https://github.com/SuffolkLITLab/docassemble-ALGenericJurisdiction).
+1. `categories` is the [LIST taxonomy](https://taxonomy.legal/) code for this interview, which can be used by your organization to organize your interviews.
+1. `attachment block variable` used to be used in the code that sends documents to courts. It's now deprecated in favor of the [ALDocument object block](#aldocument-object-block)
+1. `logic block variable` is also deprecated
+1. `typical role`: controls which questions the user gets asked, and serves the same purpose as the code in the [interview order block](#interview-order)
+
 ## Main intro page
-Adds this text to the organization's intro page that appears at the beginning of every interview. This lets your user know right away that they have gotten to the right form (or the wrong one).
+Adds this text to the organization's intro page that appears at the beginning of every interview. This lets your user know right away that they have gotten to the right (or wrong) form. Note that this can (and should) be different from the short title in the [metadata block](#metadata)
```

Comment:
```suggestion
Adds this text to the organization's intro page that appears at the beginning of every interview. This lets your user know right away that they have gotten to the right (or wrong) form. Note that this should often be more descriptive than the short title in the [metadata block](#metadata).
```

Reply:
Want to know in what way should it be different.

Reply:
Good point. I think we wanted to specifically go for action oriented titles, so I mentioned that as well.